dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -736,11 +736,11 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON 0, IDC_LINK_PROBLEM_ICON, 5, 5, 0, 0
-    LTEXT "Elementul '%s' la care această scurtătură face referire a fost schimbată sau mutată, așa că scurtătura nu va mai funcționa corect.", IDC_LINK_PROBLEM_LABEL1, 35, 5, 210, 35
-    LTEXT "Cea mai apropiată potrivire, bazată pe nume, tip, mărime și dată:", IDC_LINK_PROBLEM_LABEL2, 35, 45, 210, 35
-    LTEXT "Reparați scurtătura pentru a atinge ținta sau doar vreți să o ștergeți?", -1, 35, 85, 210, 30
+    LTEXT "Elementul '%s' la care această scurtătură face referire a fost schimbată sau mutată, aşa că scurtătura nu va mai funcţiona corect.", IDC_LINK_PROBLEM_LABEL1, 35, 5, 210, 35
+    LTEXT "Cea mai apropiată potrivire, bazată pe nume, tip, mărime şi dată:", IDC_LINK_PROBLEM_LABEL2, 35, 45, 210, 35
+    LTEXT "Reparaţii scurtătura pentru a atinge ţinta sau doar vreţi să o ştergeţi?", -1, 35, 85, 210, 30
```

Comment:
notice the added i -> ii. Is that correct?

Reply:
No, it isn't. It must be only a single i.

Reply:
Thank you!
```suggestion
    LTEXT "Reparaţi scurtătura pentru a atinge ţinta sau doar vreţi să o ştergeţi?", -1, 35, 85, 210, 30
```

Reply:
You're welcome!